emukit/quadrature/interfaces/base_gp.py

```diff
@@ -13,6 +13,12 @@ class IBaseGaussianProcess(IModel):
     """Interface for the quadrature base-GP model"""
 
     def __init__(self, kern: QuadratureKernel) -> None:
+        """
+        If this GP is initialized with data, use the raw evaluations Y of the integrand and not transformed values as
+        this is a general class that can be used with various quadrature methods. The transformation will be performed
```

Comment:
I can't follow this sentence. Can it be simpler?

Reply:
I think I can simply just say 'If this GP is initialized with data X, Y, use the raw evaluations Y of the integrand' and skip the explanation? It might cause more confusion that it solves.